pkg/ingester/lifecycle_test.go

```diff
@@ -123,8 +120,11 @@ func TestIngesterTransfer(t *testing.T) {
 		}, nil
 	}
 
-	// Now stop the first ingester
+	// Now stop the first ingester, and wait for the second ingester to become ACTIVE.
 	ing1.Shutdown()
+	poll(t, 10*time.Second, ring.ACTIVE, func() interface{} {
```

Comment:
It seems like this poll is very necessary. Was this test just passing by luck beforehand?

Reply:
Ohh wait, looks like Shutdown will block until all the chunks are transferred.

Reply:
Yes not very necessary, just there when I was debugging.  Can remove if you like?

Reply:
It seems like a reasonable check to me, I would say leave it in.